typescript/cli/src/hook/read.ts

```diff
@@ -35,7 +37,9 @@ export async function readHookConfig({
       break;
     }
     default: {
-      const provider = await context.altVmProvider.get(chain);
```

Comment:
What if we need this provider more than once? Should we create it once and store it in the context? Or is the moment exactly right here and in other CLI command code?

Reply:
added in signerMiddleware, which runs after contextMiddleware